fcli-core/fcli-ssc/src/main/resources/com/fortify/cli/ssc/actions/zip/bulkaudit.yaml

```diff
@@ -0,0 +1,328 @@
+# yaml-language-server: $schema=https://fortify.github.io/fcli/schemas/action/fcli-action-schema-2.1.0.json
+
+author: Fortify
+usage:
+  header: (PREVIEW) Perform SAST Aviator audits of SSC application versions in bulk.
+  description: |
+    This action identifies SSC application versions with pending audit issues and
+    automatically submits them for auditing by Fortify SAST Aviator. The action
+    intelligently filters to only process versions with actual pending audit issues,
+    while ignoring versions for which audit information needs to be refreshed.
+
+    For application versions that don't already exist in Aviator, the action will
+    automatically create them before submitting for audit.
+
+    Either --tag-mapping or --add-aviator-tags must be specified. The default tag
+    mapping leaves unsure issues unaudited, causing indefinite reselection. Use
+    a custom mapping file or 'fcli ssc aviator prepare' to configure Aviator-specific
+    issue templates.
+
+    This action assumes active sessions with SSC, Aviator (user role for auditing),
+    and Aviator (admin role for application listing and creation).
+
+config:
+  output: immediate
+  rest.target.default: ssc
+  run.fcli.status.log.default: true
+  run.fcli.status.check.default: false
+
+cli.options:
+  max-audits:
+    names: --max-audits, -m
+    description: "Maximum number of project versions to audit. Default: -1 (unlimited)"
+    required: false
+    default: -1
+    type: int
+  filter:
+    names: --filter, -f
+    description: "Optional filter to apply when querying SSC for projects. Example: 'Languages:java'. Default: no filtering"
+    required: false
+    default: ""
+  exclude-filter:
+    names: --exclude-filter, -e
+    description: "Optional inverse filter to exclude projects from audit. Projects matching this filter will be skipped. Example: 'Languages:c#' to exclude .NET projects. Can be combined with --filter. Default: no exclusion"
+    required: false
+    default: ""
+  dry-run:
```

Comment:
Dry run adds quite a lot of complexity in action implementation, and needs to be explicitly tested. I understand the reason, but is there anything that we can do to make this easier? 

For example, would be be beneficial to have generic support in fcli actions for dry runs? Any suggestions as to what that should look like? For example, maybe we could have a new `run.fcli::skip.if-dry-run` instruction that checks whether the `dry-run` CLI option has been specified, and if so, just outputs something like `Would run: <fcli cmd>`? Any other ideas?